### DISCUSSION TITLE
Disable Window Keymaps

### DISCUSSION THREAD

**Discussion by MadeByJay** [LazyVim/LazyVim#851] [Q&A] [answered]
Hello, how do I disable all window keymaps? They're clashing with both my own custom keymaps and a couple plugin keymaps.

Also, how could I disable other general keymaps?

**Comment by folke** [comment, marked as answer]
You can disable loading of the lazyvim keymaps. See here https://www.lazyvim.org/configuration#default-settings

Alternatively, you can delete the keymaps you don't want in your `lua/config/keymaps.lua`

**Reply by MadeByJay** [reply]
Thanks for the reply folke! Awesome work!

I think selectively deleting keymaps is what I'm looking for. Is that a lua/neovim thing? I don't know the pattern/convention for that currently. I think that's what I've been looking for in terms of the general keymaps. Thanks for the disable all tho, might A/B some keymaps when I learn the what-is.
